### PR TITLE
[Doc] 3.1 MV Updates

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: docs/sql-reference/sql-statements/data-definition/ALTER MATERIALIZED VIEW.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This SQL statement can：
 - Alter the name of an asynchronous materialized view.
 - Alter the refresh strategy of an asynchronous materialized view.
 - Alter the status of an asynchronous materialized view to active or inactive.
+- Perform atomic exchange between two asynchronous materialized views.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/ALTER MATERIALIZED VIEW.md:10]: Perform an atomic swap??

File: docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md [hunk 2]
```diff
@@ -203,11 +226,12 @@ Properties of the asynchronous materialized view. You can modify the properties
...
-- `mv_rewrite_staleness_second`: If the materialized view's last refresh is within the time interval specified in this property, this materialized view can be used for query rewrite. Otherwise, this materialized view is considered stale and not the candidate materialized view for query rewrite. Unit: Second. This property is supported from v3.0.
+- `mv_rewrite_staleness_second`: If the materialized view's last refresh is within the time interval specified in this property, this materialized view can be used directly for query rewrite, regardless of whether the data in the base tables changes. If the last refresh is before this time interval, StarRocks checks whether the base table have been updated to determine whether the materialized view can be used for query rewrite. Unit: Second. This property is supported from v3.0.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-definition/CREATE MATERIALIZED VIEW.md:234]: pluralism：StarRocks checks whether the base tables --- tables

File: docs/using_starrocks/Materialized_view.md [hunk 3]
```diff
@@ -18,12 +18,11 @@ Additionally, asynchronous materialized views are especially useful for building
 
 StarRocks v2.3 and earlier versions provided a synchronized materialized view that can be built only on a single table. Synchronized materialized views, or the Rollup, retain higher data freshness and lower refreshing costs. However, compared to asynchronous materialized views supported from v2.4 onwards, synchronized materialized views have many limitations. You have limited choices of aggregation operators when you want to build a synchronized materialized view to accelerate or rewrite your queries.
 
-The following table compares the asynchronous materialized views (ASYNC MVs) in StarRocks v2.5, v2.4, and the synchronized materialized view (SYNC MV) in the perspective of features that they support:
+The following table compares the asynchronous materialized views (ASYNC MV) and the synchronized materialized view (SYNC MV) in StarRocks in the perspective of features that they support:
```
Comment (evelynzhaojie) [inline docs/using_starrocks/Materialized_view.md:21]: the synchronized materialized view --- synchronous

**Review by hellolilyliuyi — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**